vim_binding.js

```diff
@@ -119,6 +119,30 @@ define([
         }
       }
   };
+
+  // soft-failing shortcut helper - works around the hard-failing shortcut
+  // manager in IPython
+  var add_shortcuts = function add_shortcuts(manager_name, data){
```

Comment:
It doesn't mess the global namespace - it is just the function name which might be useful for debugging.
Try `var bar = function foo(){};` in your console ;-)